ui/package.json

```diff
@@ -6,7 +6,7 @@
         "src"
     ],
     "scripts": {
-        "build": "rm -Rf dist && NODE_OPTIONS='' NODE_ENV=production webpack --mode production --config ./src/app/webpack.config.js",
+        "build": "rm -Rf dist && NODE_OPTIONS='--openssl-legacy-provider' NODE_ENV=production webpack --mode production --config ./src/app/webpack.config.js",
         "start": "NODE_OPTIONS='--no-experimental-fetch' webpack-dev-server --config ./src/app/webpack.config.js",
         "lint": "tslint --fix -p ./src/app",
         "test": "jest"
```

Comment:
on line 15 below, `engines.node` should be updated as well

Reply:
let me add that.